.gitignore

```diff
@@ -4,3 +4,4 @@
 /.cache
 /.coverage
 /.tox
+pygments_ansi_color.egg-info
```

Comment:
I'd recommend you add `*.egg-info` to your global gitignore instead of changing this one

Reply:
That would be a mistake because it would mean that every potential user/contributor would face the same problem and they will have to customize their git config to avoid it. It would just be a "bad by default" problem. It would only make contribution harder.

PS. I do use global ignores as this kind of mistake is quite common. There is also a specific case where they are the right answer: IDE settings, like ".vscode" folder, as you do not want to endup with contribution for each user IDE.

Still, these folder is a result of running project own testing pipelines and not a 3rd party tool. That makes it a minor build bug.

Reply:
Would you find add in `*.egg-info` better?